src/Services/QueuedJobService.php

```diff
@@ -2,27 +2,27 @@
 
 namespace Symbiote\QueuedJobs\Services;
 
-use Exception;
-use Psr\Log\LoggerInterface;
-use SilverStripe\Control\Controller;
-use SilverStripe\Control\Director;
-use SilverStripe\Control\Email\Email;
-use SilverStripe\Core\Config\Config;
-use SilverStripe\Core\Config\Configurable;
-use SilverStripe\Core\Convert;
-use SilverStripe\Core\Extensible;
-use SilverStripe\Core\Injector\Injectable;
-use SilverStripe\Core\Injector\Injector;
-use SilverStripe\ORM\DataList;
+use Symbiote\QueuedJobs\Tasks\Engines\TaskRunnerEngine;
+use Symbiote\QueuedJobs\QJUtils;
+use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
+use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Security\Security;
+use SilverStripe\Security\Member;
+use SilverStripe\ORM\FieldType\DBDatetime;
 use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\DataList;
 use SilverStripe\ORM\DB;
-use SilverStripe\ORM\FieldType\DBDatetime;
-use SilverStripe\Security\Member;
-use SilverStripe\Security\Security;
-use SilverStripe\Subsites\Model\Subsite;
-use Symbiote\QueuedJobs\DataObjects\QueuedJobDescriptor;
-use Symbiote\QueuedJobs\QJUtils;
-use Symbiote\QueuedJobs\Tasks\Engines\TaskRunnerEngine;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Core\Extensible;
+use SilverStripe\Core\Convert;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Controller;
+use Psr\Log\LoggerInterface;
+use Exception;
```

Comment:
It looks like your IDE is set to order imports in reverse (`Z-a`). Super minor request but do you mind reverting this part of your commit?

Reply:
Decsendist! 😛